File: Backend/Domain/Models/Digimons/Attributes.cs
namespace Backend.Domain.Models.Digimons
{
    public record class Attributes
    {
        public int Strength { get; set; }
        public int Defense { get; set; }
        public int Spirit { get; set; }
        public int Wisdom { get; set; }
        public int Speed { get; set; }
        public int Charisma { get; set; }
    }
}

File: Frontend/src/models/attributes.ts
import type { DigimonStatus } from './digimon-status';

export interface Attributes {
    strength: DigimonStatus;
    defense: DigimonStatus;
    spirit: DigimonStatus;
    wisdom: DigimonStatus;
    speed: DigimonStatus;
    charisma: DigimonStatus;
}
